jaffle_shop/models/staging/src_seed/_models.yml

```diff
@@ -1,12 +1,30 @@
 version: 2
 
 models:
+  - name: stg_customers_pii
+    columns:
+      - name: customer_id
+        tests:
+          - unique
```

Comment:
Something missing here to put _pii model in `consumer_sensitive`

Reply:
```suggestion
    meta:
      sensitive: true
    columns:
      - name: customer_id
        tests:
          - unique
```

Reply:
👍